parse_test.go

```diff
@@ -72,6 +72,39 @@ func TestParseUntilEOF(t *testing.T) {
 	}
 }
 
+// TestParse_CEchoRQ is a test for parsing a CEchoRQ command. It checks that
+// the command is parsed correctly and that the expected tags are present in
+// the dataset.
+func TestParse_CEchoRQ(t *testing.T) {
+	commandBytes := []byte{
```

Comment:
Can we try bootstrapping this test with a very simple single element? I don't love super opaque raw bytes in tests since they're hard to understand what the set up is. With a simple annotated example it might be a bit clearer and still test what we need to e.g.

```
data := []byte{
    0x0000, // Tag Group
    0x0000, // Tag Element
    ... // VR, Value
}
```

What do you think?